packages/lib/src/interactions/RatioSelectionRect.tsx

```diff
@@ -15,22 +15,19 @@ function RatioSelectionRect(props: Props) {
   const { startPoint, endPoint, ratio, ...svgProps } = props;
   const { visSize, dataToWorld, worldToData } = useVisCanvasContext();
 
-  const [dataStartPoint, dataEndPoint] = getRatioRespectingRectangle(
-    startPoint,
-    endPoint,
-    ratio
-  ).map(dataToWorld);
+  const [ratioWorldStartPoint, ratioWorldEndPoint] =
+    getRatioRespectingRectangle(startPoint, endPoint, ratio).map(dataToWorld);
```

Comment:
Woops, I mixed things up in https://github.com/silx-kit/h5web/commit/959cfe96bac3fcab0e925134f7062d04afe5d559#diff-64ebff252b1278ccdd5df1c8180f14172dba22e526031abde6ae8d0283108e51. The points go through `dataToWorld`, so they are world points here and data points below... I've renamed again and added prefixes back for clarity.